src/test/java/store/controller/ProductControllerTest.java

```diff
@@ -26,76 +26,83 @@
 
 @RunWith(MockitoJUnitRunner.class)
 public class ProductControllerTest {
-    private MockMvc mockMvc;
     @Mock
     private ProductService productService;
     @InjectMocks
     private ProductController productController;
     private static ProductDTO mockProduct;
 
     @Before()
-    public void setUp() throws JsonProcessingException {
-        MockitoAnnotations.initMocks(this);
-        mockMvc = MockMvcBuilders.standaloneSetup(productController).build();
+    public void setUp() {
         mockProduct = new ProductDTO("Test Product", 100, "Test Description");
-        mockProduct.setIdProduct(1L);
+        mockProduct.setId(1L);
     }
 
     @Test
-    public void testGetAllProducts() throws Exception {
+    public void testGetAllProducts(){
+        //given
```

Comment:
i wasnt clear when asked about this.
**given** block - is a block where you create some test variables to work with
**when** - contains some logic with simulation, for example when you request products from repository - you expect that database will return you product, which was created in given block
**then** - has to contain testing method call and checks for result